src/main/java/cn/nukkit/item/Item.java

```diff
@@ -1039,106 +1203,6 @@ private static void initCreativeItems() {
         addCreativeItem(Item.get(Item.SPLASH_POTION, Potion.STRENGTH_II));
         addCreativeItem(Item.get(Item.SPLASH_POTION, Potion.WEAKNESS));
         addCreativeItem(Item.get(Item.SPLASH_POTION, Potion.WEAKNESS_LONG));
-
-        addCreativeItem(Item.get(Item.COAL, 0));
```

Comment:
Why here was removed... In my opinion, there is no problem, because MCPE also added these items to creative inventory...